src/router/index.ts

```diff
@@ -16,6 +17,7 @@ const Router = express.Router()
 Router.use('/login', login)
 Router.use('/users', isAuthorized, users)
 Router.use('/status', status)
+Router.use('/user-course', userCourse)
```

Comment:
Should this be an authorized router? I don't think we don't want unauthorized people getting anything that's in here.

Reply:
Updated in commit